src/Aspire.ProjectTemplates/cgmanifest.json

```diff
@@ -0,0 +1,141 @@
+{
+    "$schema": "https://json.schemastore.org/component-detection-manifest.json",
+    "version": 1,
+    "registrations": [
+        {
+            "component": {
+                "type": "nuget",
+                "nuget": {
+                    "name": "OpenTelemetry.Exporter.OpenTelemetryProtocol",
+                    "version": "1.9.0"
+                }
+            }
+        },
+        {
+            "component": {
+                "type": "nuget",
+                "nuget": {
+                    "name": "OpenTelemetry.Extensions.Hosting",
+                    "version": "1.9.0"
+                }
+            }
+        },
+        {
+            "component": {
+                "type": "nuget",
+                "nuget": {
+                    "name": "OpenTelemetry.Instrumentation.AspNetCore",
+                    "version": "1.9.0"
+                }
+            }
+        },
+        {
+            "component": {
+                "type": "nuget",
+                "nuget": {
+                    "name": "OpenTelemetry.Instrumentation.Http",
+                    "version": "1.9.0"
+                }
+            }
+        },
+        {
+            "component": {
+                "type": "nuget",
+                "nuget": {
+                    "name": "OpenTelemetry.Instrumentation.Runtime",
+                    "version": "1.9.0"
+                }
+            }
+        },
+        {
+            "component": {
+                "type": "nuget",
+                "nuget": {
+                    "name": "Microsoft.Extensions.Http.Resilience",
+                    "version": "8.6.0"
+                }
+            }
+        },
+        {
+            "component": {
+                "type": "nuget",
+                "nuget": {
+                    "name": "Microsoft.Extensions.ServiceDiscovery",
```

Comment:
This is built from dotnet/aspire. Should it be here?

Reply:
This is used directly from the templates, so I think this should be added just because we know it's a direct dependency. Unless we say we already use it so we don't have to add it. IMO it doesn't hurt to keep it there.

https://github.com/dotnet/aspire/blob/main/src/Aspire.ProjectTemplates/templates/aspire-empty/AspireApplication.1.ServiceDefaults/AspireApplication.1.ServiceDefaults.csproj#L14

Reply:
Having it here will flag if somehow the templates do not point at the latest version. (Presumably because our script has a problem?)

Reply:
I am currently trying to update the files such that we don't have static versions in the project templates but use Directory.Packages.props

Reply:
Couldn't find a good way to centrally manage it so only removing this dependency for now.
It's part of the solution to there is no reason to add it to cgmanifest, we don't set the version manually, it's automated on build so it will never be a different version.